### PR TITLE
Add support to print QTensor in cpp

### PR DESCRIPTION
**Pull request by supriyar**
Summary: Print quantized tensor by first dequantizing it and then printing. Also print the scale, zero_point. size and type of tensor.

Differential Revision: D16286397

### CODE REVIEW

File: torch/nn/quantized/__init__.py [hunk 1]
```diff
@@ -1,4 +1,2 @@
-
-from __future__ import absolute_import, division, print_function, unicode_literals
-
-from .modules import *  # noqa: F401
+from . import functional  # noqa: F401
```
Comment (jerryzh168) [inline torch/nn/quantized/__init__.py]: what is this change? revert?

Comment (supriyar) [reply torch/nn/quantized/__init__.py]: Not sure how this got exported. I'll revert

File: aten/src/ATen/core/Formatting.cpp [hunk 2]
```diff
@@ -238,7 +238,17 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
     stream << "size:\n" << tensor_.sizes() << "\n";
     stream << "]";
   } else {
-    Tensor tensor = tensor_.to(kCPU, kDouble).contiguous();
+    Tensor tensor;
+    if (tensor_.is_quantized()) {
```
Comment (jerryzh168) [inline aten/src/ATen/core/Formatting.cpp]: let's discuss here, so what about toString?

Comment (supriyar) [reply aten/src/ATen/core/Formatting.cpp]: toString only prints the type of the tensor. So in this case it prints `QuantizedCPUQUInt8Type{}`

Comment (jerryzh168) [reply aten/src/ATen/core/Formatting.cpp]: oh looks like it only gives the type of Tensor?

File: aten/src/ATen/core/Formatting.cpp [hunk 3]
```diff
@@ -238,7 +238,17 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
...
+      tensor = tensor_.dequantize().to(kCPU, kDouble).contiguous();
+      stream << "[ " << tensor_.toString() << "{}\n";
+      stream << "size: " << tensor_.sizes() << "\n";
+      stream << "scale: " << tensor_.q_scale() << "\n";
+      stream << "zero_point: " << tensor_.q_zero_point() << " ]\n";
+    }
+    else {
+      tensor = tensor_.to(kCPU, kDouble).contiguous();
+    }
```
Comment (jerryzh168) [inline aten/src/ATen/core/Formatting.cpp]: I think probably `__printTensor`? and please match the format with float Tensor.

**Pull request closed without merging**